nsv13/code/game/objects/structures/large_stairs.dm

```diff
@@ -0,0 +1,18 @@
+/**
+*
+* Stairs that behave like stairs actually should because /tg/ stairs are a pile of ass.
+* Just place stairs facing up and down above each other.
+* These stairs only go up one level. If you stack a tonne of them you won't have a good time
+* Make sure theyre facing the same direction or it'll feel weird traversing them.
+* MAPPERS: Please enclose your stairs with walls. Otherwise theyll feel weird.
+*
+*
+*/
+
+/obj/structure/stairs/large_stairs
```

Comment:
This should also be in another PR, but i'll allow it to piggyback off of Enterprise.